PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceComboBoxModel.java

```diff
@@ -25,10 +25,12 @@
 import com.microsoft.azure.management.appservice.WebAppBase;
 import com.microsoft.azuretools.core.mvp.model.ResourceEx;
 import lombok.Getter;
+import lombok.Setter;
 import org.apache.commons.lang3.StringUtils;
 
 @Getter
 public class AppServiceComboBoxModel<T extends WebAppBase> {
+    @Setter
     protected boolean isNewCreateResource;
```

Comment:
usually bool field name should be adj. how about `new`/`creating`?
https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter